src/class-parsely.php

```diff
@@ -793,6 +796,29 @@ public function display_admin_warning() {
 		<?php
 	}
 
+	/**
+	 * Display a dismissible admin warning if the current WordPress or PHP versions are below the required minimum for the 3.0 release of wp-parsely
+	 * We should get rid of this warning when we release 3.0
+	 *
+	 * @category Function
+	 * @package Parsely
+	 */
+	public function display_admin_upgrade_warning() {
+		global $wp_version;
+		if ( version_compare( phpversion(), '7.1', '>=' ) && version_compare( $wp_version, '5.0', '>=' ) ) {
```

Comment:
```suggestion
		if ( version_compare( PHP_VERSION, '7.1.0', '>=' ) && version_compare( $wp_version, '5.0', '>=' ) ) {
```

A constant is more performant than a function call.